packages/charting/index.html

```diff
@@ -1,58 +1,54 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Heh, we should also be specifying `lang="en"` for all our documents (I think). Not for you to fix tho.